File: app/src/main/java/com/remember/app/ui/splash/SplashPresenter.java
package com.remember.app.ui.splash;

import com.arellomobile.mvp.InjectViewState;
import com.remember.app.Remember;
import com.remember.app.ui.base.BasePresenter;

@InjectViewState
public class SplashPresenter extends BasePresenter<SplashView> {

    SplashPresenter() {
        Remember.getApplicationComponent().inject(this);
    }

}

File: app/src/main/java/com/remember/app/ui/splash/SplashPresenter.kt
package com.remember.app.ui.splash

import com.arellomobile.mvp.InjectViewState
import com.remember.app.Remember
import com.remember.app.ui.base.BasePresenter

@InjectViewState
class SplashPresenter : BasePresenter<SplashView>() {

    init {
        Remember.applicationComponent.inject(this)
    }

}
